src/app/store.ts

```diff
@@ -1,6 +1,15 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
+import { currentTodoSlice } from '../features/currentTodo';
+import { filterSlice } from '../features/filter';
+import { todosSlice } from '../features/todos';
+import { loaderSlice } from '../features/loader';
 
-const rootReducer = combineSlices();
+const rootReducer = combineSlices(
+  currentTodoSlice,
+  filterSlice,
+  todosSlice,
+  loaderSlice,
+);
```

Comment:
Replace `combineSlices` with `combineReducers` to correctly combine the slices into a root reducer.